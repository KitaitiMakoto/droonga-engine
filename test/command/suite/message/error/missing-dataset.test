#@include fixture/user-table.jsons
#@disable_validation
{
  "type": "add",
  "body": {}
}
