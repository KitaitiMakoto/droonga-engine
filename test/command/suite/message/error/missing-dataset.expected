[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 400,
    "type": "add.result",
    "body": {
      "name": "MissingDatasetParameter",
      "message": "\"dataset\" must be specified."
    }
  }
]
